openmc/cell.py

```diff
@@ -23,7 +23,7 @@ def reset_auto_cell_id():
 
 
 class Cell(object):
-    """A region of space defined as the intersection of half-space created by
+    r"""A region of space defined as the intersection of half-space created by
```

Comment:
Are we missing these everywhere? Or is this simply needed if math is embedded in the docstring?